docs/execution.rst

```diff
@@ -0,0 +1,210 @@
+is available in this SPIE Paper :cite:`DMTN-140`.  
+
+
+
+Test Specification Generation
+#############################
+
+Test specifications are extracted using the Jira REST API.
+All tests cases included in a TM4J folder, including subfolders, are rendered in the same extraction.
+The folder organization in Jira should correspond to the major subsystem components and the MagicDraw model.
+
+The syntax to extract a test specification is the following::
+
+  docsteady generate-spec "</tm4j_folder>" jira_docugen.tex
+
+where ``</tm4j_folder>`` shouldlbe replaced by the exact folder where the test cases are defined in the Jira ``Test Cases`` tab.
+For example, the command to extract the DM Acceptance test specification, LDM-639, is the following:
+
+``docsteady generate-spec "/Data Management/Acceptance|LDM-639" [jira_docugen.tex]``
+
+Note that:
+- the output file \textit{jira\_docugen.tex`` is optional in the execution of docsteady but required in this context in order to include the extracted information in a \LaTeX~document (i.e. LDM-639.tex). If omitted, the docsteady output will just be printed in the terminal;
+- the folder name in Jira includes at the end the test specification document handle. This is a best practice to use when organizing test cases in Jira since it helps to orient the user in the folder structure;
+- an appendix with the traceability to the requirements is produced in the file `jira_docugen.appendix.tex`, to be included in the test specification TeX file.
+
+
+See LDM-639 Git repository as an example.
+
+
+.. _tprg:
+
+Test Plan and Report Generation
+###############################
+
+This is automated - see :ref:`Test plan report with SQRBOT-JR<tprsqrbot>`.
+
+``Important``: before extracting a test plan and report using docsteady,
+the corresponding document handle has to be added in the ``Document ID`` field in the Jira test plan object.
+This ensures that the Verification Control Document will include this information.
+
+The following command extracts a Test Plan and Report using Jira REST API:
+
+``docsteady generate-tpr <LVV-PXX> <file.tex> [--trace true]``
+
+Where:
+
+\begin{itemize``
+\item ``LVV-PXX`` is the TM4J object that describes the test campaign, for example ``LVV-P72``;
+\item ``file.tex`` is the Test Plan and Report tex file where the document will be rendered, for example ``DMTR-231.tex``;
+\item ``--trace true`` (optional) generates an appendix with the traceability information.
+\end{itemize``
+
+Each TM4J test plan and related information in Jira is rendered in a different Test Plan and Report document,
+the filename of which usually corresponds also to the document handle in DocuShare.
+
+The generated file can be built directly into the corresponding pdf, however additional files are required:
+
+- Makefile
+- appendix.tex
+- history\_and\_info.tex
+
+When creating the Git repository using ``sqrbot-jr``, all the required files should already be present.
```

Comment:
Note that SQR-006 is not relevant to sqrbot-jr. The best link is to: https://developer.lsst.io/project-docs/change-controlled-docs.html#ccd-create